src/client/app/project/ProjectComponent.ts

```diff
@@ -55,5 +66,6 @@ export class ProjectComponent implements OnInit{
 	public set project(project: Project) {
 		this._project = project;
 		this.projectService.project = project;
+		this.symbolService.parseProject(this._project);
```

Comment:
@jangmarker I suggest to create an event in the ProjectService "projectChanged" and DI it in the symbolService